### ISSUE TITLE
React/RCTBridgeModule.h file not found

### ISSUE THREAD

**Issue by bitsal** [react-native-config/react-native-config#86]
react-native: 0.40.0
react-native-config: 0.3.1
react-native-cli: 2.0.1
XCode: Version 8.1 (8B62)
CocoaPods: 1.2.0

I've configured everything based on the README.
(the project is opened *.xcworkspace in XCode)
1. `react-native-config` was linked
![screen shot 2017-02-21 at 9 40 16 pm](https://cloud.githubusercontent.com/assets/5635839/23179372/ad590602-f87e-11e6-9238-a00cdec99ba1.png)
![screen shot 2017-02-21 at 9 43 06 pm](https://cloud.githubusercontent.com/assets/5635839/23179412/dc87c350-f87e-11e6-8d4a-0902d5b25592.png)
![screen shot 2017-02-21 at 9 44 31 pm](https://cloud.githubusercontent.com/assets/5635839/23179497/2acd9a80-f87f-11e6-8230-92a2804ec358.png)

2. Product -> Clean
3. Product -> Build -> Failed
<img width="253" alt="screen shot 2017-02-21 at 9 48 22 pm" src="https://cloud.githubusercontent.com/assets/5635839/23179630/9b4a9768-f87f-11e6-8611-00533089c9f2.png">

Could help me to understand why it doesn't work and what is the root cause?
Thank you in advance!

**Comment by bitsal** [comment]
Any ideas?
Any help is appreciated.

**Comment by KazuyaHara** [comment]
@bitsal 
Do you list 'React' on Build > Targets before application?

![targets](https://cloud.githubusercontent.com/assets/8471513/23313564/960d843e-fb01-11e6-928e-5aea3fb5ec70.png)

If you created new scheme, your application might be there but React isn't as default.

**Comment by bitsal** [comment]
@KazuyaHara 
Yes, It is there. I've already read about such fix and one more like disabling parallel builds.

**Comment by peterept** [comment]
I'm also seeing the same error.

**Comment by bitsal** [comment]
@peterept does it work with React Native >= 0.40.0?

**Comment by bitsal** [comment]
Guys, it really blocks us using new react-native `> 0.40.x`. I wonder if it works for someone with new react native.

**Comment by eugenehp** [comment]
@bitsal have you found a workaround for this using RN 0.40x ?

**Comment by bitsal** [comment]
@eugenehp No, we've eliminated react-native-config from the project in favor of custom approach due to lack of contribution. It seems only one guy is a real contributor here, it's not convenient for us waiting for the response and real help for ages.

**Comment by bitsal** [comment]
FYI,
I don't know if it helps (we stopped using react-native-config).
If you use Cocoapods some of the libraries are linked incorrectly:
Libraries that you link via Cocoapods still should be under `Libraries` folder but not in linking section (I mean e.g. `libReact.a`).

E.g. some of my issues have gone when I:

1. added React.xcodeproj under Libraries
2. added React to the schema -> Build -> targets (BUT not from Pods, only from React.xcodeproj)
3. removed libReact.a from Build Phases -> Link Binary With Libraries section
